### DISCUSSION TITLE
How to add command-line args?

### DISCUSSION THREAD

**Discussion by agriffis** [jellydn/hurl.nvim#174] [Q&A] [answered]
I need to call hurl with `--cacert FILENAME`. Is there a way to configure `hurl.nvim` for that?

**Comment by agriffis** [comment]
For the moment, I just wrapped hurl:

```
#!/bin/bash
exec /usr/bin/hurl --cacert FILENAME "$@"
```

**Reply by jellydn** [reply, marked as answer]
You could add as option after `Hurl` command, it will send as the extra param.

**Reply by agriffis** [reply]
Hmm, thanks, I'll have to experiment with my workflow to see what works best, but that is good to know.
